examples/camera-app/camera-common/include/camera-device-interface.h

```diff
@@ -126,6 +127,9 @@ class CameraDeviceInterface
     // Getter for the Media Controller
     virtual MediaController & GetMediaController() = 0;
 
+    // Setter for the Camera AV Stream Management Server
+    virtual void SetAVStreamMgmtServer(CameraAVStreamMgmtServer * cameraAVStreamMgmtServer) = 0;
```

Comment:
Why do we need this Setter. If it is the CameraAVStreamMgmtServer that is needed, we can use the GetCameraAVStreamMgmtDelegate() and invoke a function in that delegate.

Reply:
But CameraAVStreamMgmtDelegate does not provide the info we needed, those info are only available in CameraAVStreamMgmtServer